checkout/checkin.md

```diff
@@ -259,6 +259,16 @@ the following argument objects:
 }
 ```
 
+{:.code-header}
+**Billing Details Available Event Argument Object**
+
+```js
+{
+    "actionType": "OnBillingDetailsAvailable",
+    "url": "https://api.externalintegration.payex.com/psp/consumers/<consumerProfileRef>/billing-details"
```

Comment:
Do we have examples of what a `GET` request towards `https://api.externalintegration.payex.com/psp/consumers/<consumerProfileRef>/billing-details` returns that we can show?

Reply:
Got this from Syver (check last commit). Is this something in the area of what you had in mind?

Reply:
Perfect! Similar examples for all other URLs provided in events would be great, but can be fixed as a part of DX-472 in a separate branch.

Reply:
Great! Let's wrap this one up :)